wlauto/workloads/gmail/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -97,6 +100,10 @@ public void clickSendButton() throws Exception {
         clickUiObject(sendButton, timeout);
         result.end();
         timingResults.put("Send", result);
+
+        UiObject drawerButton = new UiObject(new UiSelector().descriptionContains("avigat")
```

Comment:
It is not clear what "avigat" is meant to find, presumably a variant of 'Navigation' ?